<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-LAK4O3A" failures="1" tests="4" name="Test" time="0.015" errors="0" timestamp="2024-03-07T19:27:49 EST">
  <testcase classname="testPackage.Day2" name="Test1" time="0.001"/>
  <testcase classname="testPackage.Day2" name="Test1" time="0.0"/>
  <testcase classname="testPackage.Day2" name="Test1" time="0.0"/>
  <testcase classname="testPackage.Day2" name="Test2" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testPackage.Day2.Test2(Day2.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
  <testcase classname="testPackage.Day2" name="Test3" time="0.0">
    <ignored/>
  </testcase> <!-- Test3 -->
</testsuite> <!-- Test -->
